.github/workflows/params.yml

```
@@ -1,7 +1,7 @@
 name: TEC Params Call
 on:
   schedule:
-    - cron:  '0 14 * * 3'
+    - cron:  '0 15 * * 2'
```

Looks good to me!
I think it should be - 
```suggestion
    - cron:  '0 14 * * 2'
```
Since we would want the reminder 1 hour before the call. Right?

---

now you're thinking ahead! haha